data/json/recipes/weapon/mods.json

```diff
@@ -23,6 +23,19 @@
     ]
   },
   {
+    "result": "crafted_suppressor",
+    "type": "recipe",
+    "category": "CC_WEAPON",
+    "subcategory": "CSC_WEAPON_MODS",
+    "skill_used": "mechanics",
+    "difficulty": 1,
+    "skills_required": [ "gun", 1 ],
+    "time": 650,
+    "autolearn": true,
+    "qualities": [ { "id": "SAW_M_FINE", "level": 1 } ],
+    "components": [ [ [ "filter_liquid", 1 ], [ "filter_liquid_makeshift", 1 ] ] ]
```

Comment:
Speaking of loudness, I think I recall an issue on GitHub where someone suggested changing it to be measured in dB. There was even a rough outline found for how loud a sound needs to be heard from as far away...